libs/angular-components/src/signpost/src/signpost.component.ts

```
@@ -0,0 +1,39 @@
+import {Component, OnInit, Input, ElementRef} from '@angular/core';
+import tippy from 'tippy.js';
+
+@Component({
+  selector: 'uxg-signpost',
+  templateUrl: './signpost.component.html',
+  styleUrls: ['./signpost.component.scss'],
+})
+
+export class SignpostComponent implements OnInit {
+
+  constructor(private el: ElementRef) { }
+  tippy;
+  popper;
+
+  @Input() content;
+  @Input() position;
+  ngOnInit() {
+    this.loadTippy();
+  }
+
+  loadTippy() {
+    setTimeout(() => {
+      const tippyOptions =  {
+        content: this.content || 'Hello',
```

Should not have default content, throw an error if content is not provided.

---

Also, make sure that content can have safe html, like clickable links.